utils/frame/benchmarking-cli/src/lib.rs

```diff
@@ -72,6 +72,13 @@ pub struct BenchmarkCmd {
 	#[structopt(long)]
 	pub heap_pages: Option<u64>,
 
+	/// Run verification logic in addition to the benchmarks.
+	///
+	/// WARNING: The verification logic will be included as part of the extrinsic timing! This
+	/// should only be used for sanity checking your benchmarks are valid.
+	#[structopt(long)]
+	pub verify: bool,
```

Comment:
If I read code correctly verify doesn't have effect on the result of the benchmark:
```rust

							if verify {
								// If `--verify` is used, run the benchmark once to verify it would complete.
								repeat_benchmark(1, Default::default(), &mut Vec::new(), true)?;
							}
							repeat_benchmark(repeat, c, &mut results, false)?;
```
the results are still holding only execution without verify no ?

Then I don't see why user shouldn't expect correct result with verify

Reply:
The docs in the pr approve your interpretation :D